### ISSUE TITLE
CTRL+C except doesnt work at times

### ISSUE THREAD

**Issue by EricTalv** [EricTalv/PythonEmailScraper#4]
When the scraper is scraping
the request is made inside a try/catch block thus we can 
handle all exceptions as such.

But when it comes to printing out any data, that part isn't in a try/catcher

thus we need a better way.,

**Comment by EricTalv** [comment]
we can use `signal`

```
import signal
import sys
def signal_handler(sig, frame):
        print('You pressed Ctrl+C!')
        sys.exit(0)
signal.signal(signal.SIGINT, signal_handler)
```
By assigning `CTRL+C` a dedicated function, we can do whatever we want

**Comment by EricTalv** [comment]
Fixed commit #c1fa152
